docker/Makefile

```diff
@@ -13,6 +13,31 @@ endif
 build: .TEST
 	docker build --network host --build-arg=EMSCRIPTEN_VERSION=${version} --tag emscripten/emsdk:${version} .
 
+test: .TEST
+	# compilation as non-root
+	docker run --rm -u `id -u`:`id -g` -e HOME=/tmp -v `pwd`:/src -w /src emscripten/emsdk:${version} \
+		bash -c "\
+			mkdir -p .test \
```

Comment:
Why the `.` prefix here?   Why not just test?

Reply:
not a particular reason, just convention - I can change that.